cabal.project

```diff
@@ -20,10 +20,12 @@ packages: legacy/marconi-core-legacy
           marconi-chain-index
           marconi-core
           marconi-core-json-rpc
-          doc/read-the-docs-site
           marconi-sidechain
           marconi-sidechain-experimental
           marconi-starter
+if(!os(darwin) || impl(ghc >= 9.4))
```

Comment:
Is it known that everything works fine with ghc 9.6?

Reply:
I haven't tried that yet but I thought it would be worth re-enabling it automatically when we do upgrade. If it's permanently disabled, no-one will remember to check whether we can re-enable this. If we find we get failures again we can just increase the bounds here.